internal/xray/tree_node.go

```diff
@@ -414,11 +414,11 @@ func (t TreeNode) toTitle() (title string) {
 		}
 	}()
 
-	categ := category(t.GVR)
```

Comment:
I am Ok with docs and comments spelling but no code changes! In this case categ makes more sense to me

Reply:
fwiw, cat is a fairly common abbreviation for [category](https://writingexplained.org/english-abbreviations/category), but I'm more interested in getting some improvements in than defending any particular change.

Happy to drop any particular thing or things.